src/ui/common/src/utils/integrations.ts

```diff
@@ -230,6 +230,11 @@ export async function addTable(
 }
 
 export const SupportedIntegrations: ServiceInfoMap = {
+  ['Aqueduct']: {
```

Comment:
TODO: May want to find another way to get the engine logos, as this adds Aqueduct engine to the compute section of the Integrations page.

Perhaps we can refactor the supported integrations out into two lists: one for compute and one for data, then concatenate them into the SupportedIntegrations variable so we can omit 'aqueduct' from supported integrations.